v8/rest/guild.ts

```diff
@@ -770,3 +770,29 @@ export interface RESTPatchAPIGuildMemberVerificationJSONBody {
 }
 
 export type RESTPatchAPIGuildMemberVerificationResult = APIGuildMembershipScreening;
+
+export interface RESTPatchAPIGuildVoiceStateCurrentMemberJSONBody {
+	/**
+	 * The id of the channel the user is currently in
+	 */
+	channel_id: Snowflake;
+	/**
+	 * Toggles the user's suppress state
```

Comment:
I think this is more accurate than "toggles"
```suggestion
	 * Whether the user is suppressed from speaking
```

Reply:
I copied the description they have in the PR, and that's what's been done for forever now...even tho at times their description is..iffy

Reply:
Fair enough!